utility/utils.py

```diff
@@ -2798,3 +2799,93 @@ def is_unsecured_registry(test_data):
             return insecure_registry_value  # Return boolean value for insecure-registry
 
     return False  # Default return value if conditions are not met
+
+
+def lookup_in_directory(client, mount):
```

Comment:
We already have a lookup function.Check - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py#L674